### DISCUSSION TITLE
Allow combining floating docks

### DISCUSSION THREAD

**Discussion by KAW0** [godotengine/godot-proposals#7880] [Editor]
It would be awesome for users with multiple screens to combine floating docks into one window with tabs to avoid something like this: 
![image](https://github.com/godotengine/godot-proposals/assets/16373327/99ace97c-b48c-4221-8eab-447f342b0df9)

**Comment by kitbdev** [comment]
There's a proposal for it here https://github.com/godotengine/godot-proposals/issues/1984
